File: 351004/Bunevich/labs/task330/src/main/java/com/example/demo/dto/response/WriterResponseTo.java
package com.example.demo.dto.response;

import com.fasterxml.jackson.annotation.JsonRootName;
import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;

@Data
@JsonNaming(PropertyNamingStrategies.LowerCamelCaseStrategy.class)
@JsonRootName("writer")
public class WriterResponseTo {
    private Long id;
    private String login;
    //private String password;
    private String firstname;
    private String lastname;
}

File: 351003/Soika/entities-app/publisher/src/main/kotlin/com/example/entitiesapp/dto/WriterResponseTo.kt
package com.example.entitiesapp.dto

import java.io.Serializable

data class WriterResponseTo(
    val id: Long = 0,
    val login: String = "",
    val password: String = "",
    val firstname: String = "",
    val lastname: String = "",
    val role: Role = Role.CUSTOMER
) : Serializable